sdk/keyvault/azure-security-keyvault-jca/src/main/java/com/azure/security/keyvault/jca/KeyVaultKeyStore.java

```diff
@@ -119,19 +138,21 @@ public KeyVaultKeyStore() {
         refreshCertificatesWhenHaveUnTrustCertificate = Optional.ofNullable(System.getProperty("azure.keyvault.jca.refresh-certificates-when-have-un-trust-certificate"))
             .map(Boolean::parseBoolean)
             .orElse(false);
-        keyVaultCertificates = new KeyVaultCertificates(refreshInterval, keyVaultClient);
-        classpathCertificates = new ClasspathCertificates();
         jreCertificates = JreCertificates.getInstance();
-        allCertificates = Arrays.asList(keyVaultCertificates, classpathCertificates, jreCertificates);
+        wellKnowCertificates = new FileSystemCertificates(wellKnowPath);
+        customCertificates = new FileSystemCertificates(customPath);
+        keyVaultCertificates = new KeyVaultCertificates(refreshInterval, keyVaultClient, this);
+        classpathCertificates = new ClasspathCertificates();
+        allCertificates = Arrays.asList(jreCertificates, wellKnowCertificates, customCertificates, keyVaultCertificates, classpathCertificates);
```

Comment:
Could you please add unit test for the priority? For example:
1. wellKnowCertificates and customCertificates has same alias, the first one will be used.
2. Same to customCertificates and classpathCertificates